model.js

```diff
@@ -131,3 +131,17 @@ exports.selectUsers = () => {
       return data.rows;
     });
 };
+
+exports.deleteCommentByCommentId = (id) => {
+  return checkIfCommentExists(id).then((doesCommentExist) => {
+    if (doesCommentExist === false) {
+      return Promise.reject({ status: 404, msg: "Comment not found" });
+    } else {
+      return db
+        .query("DELETE FROM comments WHERE comment_id = $1", [id])
+        .then(() => {
+          return;
+        });
+    }
```

Comment:
I don't think you need a separate database query to check if the comment exists. I think there might be a property called `rowCount` that comes back on the resolution of the query promise that you can use to determine whether your `DELETE` actually found a comment with that id to delete.

I would investigate this value and what it looks like when you make a request to an existing comment and a non-existing comment, and I would refactor so that you don't actually need the `checkIfCommentExists` function at all!